config.c

```diff
@@ -420,6 +420,9 @@ static int pv_config_load_file(char *path, struct pantavisor_config *config)
 	config->libthttp.loglevel =
 		config_get_value_int(&config_list, "libthttp.log.level", 3);
 
+	config->libthttp.certdir = config_get_value_string(
```

Comment:
we have to free certdir and trustore in pv_config_free

Reply:
We have:

if (pv->config.libthttp.certdir)
		free(pv->config.libthttp.certdir);

Is that not enougH?

Reply:
sorry, didn't see those